f8a_worker/dispatcher/flows/stackApiGraphV2Flow.yml

```diff
@@ -12,3 +12,4 @@
           to:
             - 'recommendation_v2'
             - 'stack_aggregator_v2'
+            - 'BookkeeperTask'
```

Comment:
@fridex  Thanks for the snippet. Changed as per your comments. Will be available in the next commit.